queries.md

```diff
@@ -1,20 +1,30 @@
 # Database Queries
 
 ## find all customers that live in London. Returns 6 records.
+select * from customers where city = 'London'
 
 ## find all customers with postal code 1010. Returns 3 customers.
+select * from customers where PostalCode = 1010
 
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
+select Phone from suppliers where SupplierID = 11
 
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
+select * from orders order by OrderDate desc
 
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
+select * from suppliers where length(SupplierName) > 20
 
 ## find all customers that include the word "market" in the name. Should return 4 records.
+select * from customers where CustomerName like '%market%'
+
+// DAY 2 ?
 
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
+insert into customers values ('ID NUMBER', 'The Shire', 'Bilbo Baggins', '1 Hobbit-Hole', 'Bag End', '111', 'Middle Earth')
 
 ## update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
+update customers set PostalCode = 11122 where ContactName = 'Bilbo Baggins'
```

Comment:
yes, this one is correct (assuming the insert query prior would fire properly, which it won't yet)